sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/comsos/encryption/implementation/keyprovider/EncryptionKeyStoreProviderImpl.java

```diff
@@ -1,14 +1,15 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.cosmos.encryption.keyprovider;
+package com.azure.comsos.encryption.implementation.keyprovider;
 
 
 import com.azure.cosmos.encryption.implementation.mdesrc.cryptography.EncryptionKeyStoreProvider;
 import com.azure.cosmos.encryption.implementation.mdesrc.cryptography.KeyEncryptionKeyAlgorithm;
 import com.azure.cosmos.encryption.implementation.mdesrc.cryptography.MicrosoftDataEncryptionException;
+import com.azure.cosmos.encryption.keyprovider.EncryptionKeyWrapProvider;
```

Comment:
nit: sort by order?

Reply:
Imports are already sorted , if that you meant ?